### DISCUSSION TITLE
【权限管理】能否支持对流水线模板指定权限？

### DISCUSSION THREAD

**Discussion by whataa** [Tencent/bk-ci#4765] [Ideas]
两个问题：
1. 目前权限只能附加到”实例“，如果我的模板下的实例有1000个会累死的；（在”我的资源“中”批量添加“的功能太鸡肋，”全选“只能选当前页，一页的大小还没法修改）
2. "我的资源"的”任意“太过流氓，我想在”任意“的基础上限制不能拥有对某条流水线或某个模板的权限就不行了

**Comment by zanyzhao** [comment]
最近权限模块在做大版本升级，我们考虑下你说的场景
